docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -0,0 +1,1249 @@
+      they are explicitly declared. One of the key differences between the
+      bare metal and container/cloud is that in bare metal storage is directly
+      tied to the Thor or the Thor worker nodes or the Roxie worker nodes, or
+      even in the case of the ECLCC Server (DLLS) In containers these are
+      completely separate and anything having to do with files is defined in
+      the values.yaml.</para>
+
+      <para>Recall that every component should have a resources entry, that in
+      the delivered values.yaml file the resources are present but commented
+      out as indicated here.</para>
+
+      <para><programlisting>  #resources:
+  #  cpu: "1"
+  #  memory: "4G"
+</programlisting>The stock values file will work and allow you to stand up a
+      functional system, however you should define the component resources in
+      a manner that corresponds best to your operational strategy.</para>
+
+      <sect3 id="DALI_ValueYAML">
+        <title>Dali</title>
+
+        <para>When configuring Dali, which also has a resources section, it is
+        going to want heaps of memory and a fair bit of CPU as well. It is
+        very important to define these carefully. Otherwise Kubernetes will
+        assign all the pods to same virtual machine and components fighting
+        for memory will crush them. Therefore more memory assigned the better.
+        If you define these wrong and some process starts using too much
+        memory, Kubernetes will kill the pod if it is using much more memory
+        than what it was configured for.</para>
+      </sect3>
+
+      <sect3 id="DAFLESRV_DFURVR_YMLSECT">
+        <title>Components: dafilesvrs, dfuserver</title>
+
+        <para>components: dafilesvrs, eclccservers, dfuserver, are declared as
+        a lists in the yaml, as is the ECL Agent.</para>
+
+        <para>consider the dfuserver which is in the delivered HPCC Systems
+        values.yaml as:</para>
+
+        <programlisting>dfuserver:
+- name: dfuserver
+  maxJobs: 1</programlisting>
+
+        <para>If you were to add a mydfuserver as follows</para>
+
+        <para><programlisting>dfuserver:
+- name: dfuserver
+  maxJobs: 1
+- name: mydfuserver
+  maxJobs: 1
+</programlisting>In this scenario you would have another item here named
+        mydfuserver and it would show up in ECLWatch and you can submit items
+        to that.</para>
+
+        <para>If you wanted to add another dfuserver, you can add that to the
+        list similarly. You can likewise instantiate other components by
+        adding them to their respective lists.</para>
+      </sect3>
+
+      <sect3 id="VALYml_ECLCCServer">
+        <title>ECLCCserver</title>
+
+        <para>Some values of note for ECLCC Server section.</para>
+
+        <para><emphasis role="bold">useChildProcess</emphasis> -- primarily
+        used by ECL Agent and the ECLCC Server. When you submit a job or query
```

Comment:
it's true it's a property of eclagent too, but the heading says ECLCC Server values, so a bit confusing.
I see below the section goes on to talk about the ECL Agent and what useChildProcesses means to it.

So it sounds like the heading ("Some values of note for ECLCC Server section.") is wrong, given it's talking about the useChildProcesses and it's meaning for not just eclccserver